### PR TITLE
Add footprint form table.

### PR DESCRIPTION
**Pull request by EagleC0318**
# Why
- For issue #44 
# What
- Add trace form model/table
# How
- Add the model and run python manage.py makemigrations
- The model is referred on PR https://github.com/OH-SHOWN/ohshown-backend/pull/48/files cc @yanghaochang104 
# Test
- Check the trace form table on postgreSQL db locally
image

### CODE REVIEW

File: backend/api/models/footprint_form.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import uuid
+
+from django.db import models
+
+from .mixins import SoftDeleteMixin
+
+
+class FootprintForm(SoftDeleteMixin):
+    id = models.UUIDField(
```
Comment (EagleC0318) [inline backend/api/models/footprint_form.py:9]: Not sure which type of id is better, the auto-incremental Integer or UUID?
@tai271828 @yanghaochang104

**Comment by EagleC0318** [comment]
I am starting to create the new tables as the design in #44  @yanghaochang104
It might be also useful with the old table  @tai271828

File: backend/api/models/footprint_form.py (hunk 1 above)
Comment (tai271828) [reply backend/api/models/footprint_form.py:9]: My gut feeling is auto-incremental integer, but UUID is good for me as well.

**Review by tai271828 — COMMENTED** [review]
Overall: good! Thanks for the contribution!

I have one question and also look for @yanghaochang104 's comment (if any), but anyone please feel free to move on and land the pull request.

File: backend/api/migrations/0044_auto_20220607_1633.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0043_ohshownevent_sight_see_date_time'),
+    ]
+
+    operations = [
+        migrations.AlterField(
```
[the remark this answers is not in the document]
Comment (EagleC0318) [reply backend/api/migrations/0044_auto_20220607_1633.py:13]: All the new migrations are made by `python manage.py makemigrations` command.

**Comment by EagleC0318** [comment]
@tai271828  @Neilxx 
I updated the model based on the latest FE questionnaire with some naming difference (freshness->age, and others). Please take a look :smile: .
(Ref. https://github.com/OH-SHOWN/ohshown-backend/issues/44#issuecomment-1070730896)

**Pull request merged**